src/components/misc/personViews/cells/PersonNotesCell.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import truncText from '../../../../utils/truncText';
```

Comment:
I assume you imported this with the goal of truncating the text for long notes, the same way that `SurveyResponseCell` component truncates long responses. I think `PersonNotesCell` should too, but right now this import is not actually being used in this component, and long notes are not being truncated.

Reply:
Fixed